source/2015-06-12-panter-die-demokratische-firma.html.markdown

```diff
@@ -0,0 +1,76 @@
+---
+title: Panter die demokratische Firma
+date: 2015-06-12 08:42 UTC
+tags:
+---
+
+> Ein Lohnmodell, welches genau zwei Stufen kennt und für alle Mitarbeiter
+> transparent ist. Ein Vetorecht bei operativen Entscheide für alle Mitarbeiter.
+> Eine Unternehmensentwicklung die basisdemokratisch funktioniert. Kann das in
+> einer Firma mit mehr als 30 Mitarbeitern funktionieren?
+
+Die kurze Antwort ist: Ja, die Firma Panter funktioniert seit mehreren Jahren
+so.
+
+Doch was steckt dahinter? Warum funktioniert es? Was sind die Schattenseiten
+dieses Modells?
+
+## Im Detail
+
+Bei Panter gibt es nur zwei Lohnstufen: Senior und nicht Senior.  Abgesehen von
+dieser Abstufung verdienen alle Mitarbeitenden gleich viel (bei einem 100%
+Pensum). Der Lohn ist abhängig von den geleisteten Arbeitsstunden und der
+aktuellen Auftragslage. Die aktuellen Löhne werden jeden Monat für alle
+Mitarbeitenden ersichtlich publiziert.
```

Comment:
Ich denke die genaue Berechnung und die Hintergründe dazu könnte man in einem separaten Post erläutern.